### PR TITLE
pullRequestReviewComments

### PR DESCRIPTION
**Pull request by kisaga**
## test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
-Java API for GitHub
```
Comment (kisaga) [inline README.md:1]: This is a reply.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
+# Java API for GitHub TEST (stable)
```
Comment (maximevw) [inline README.md:2]: A single line review comment

File: README.md [hunk 3]
```diff
@@ -1,3 +1,4 @@
...
 
 See https://github-api.kohsuke.org/ for more details
```
Comment (maximevw) [inline README.md:3]: A multiline review comment

File: README.md (hunk 2 above)
Comment (maximevw) [inline README.md:2]: A single line review comment

File: README.md (hunk 3 above)
Comment (maximevw) [inline README.md:3]: A multiline review comment

**Pull request closed without merging**